app/cdap/components/WrangleHome/Components/OngoingDataExploration/index.tsx

```diff
@@ -112,10 +112,11 @@ export default function OngoingDataExploration() {
               pathname: `/ns/${getCurrentNamespace()}/wrangler-grid/${`${item[4].workspaceId}`}`,
               state: {
                 from: T.translate('features.WranglerNewUI.Breadcrumb.labels.wrangleHome'),
-                path: T.translate('features.WranglerNewUI.Breadcrumb.params.wrangeHome'),
+                path: T.translate('features.WranglerNewUI.Breadcrumb.params.wrangleHome'),
```

Comment:
`wranglerHome` instead of `wrangleHome`?

Reply:
Updated.